public/js/utils/source-editor.js

```diff
@@ -4,10 +4,7 @@ require("codemirror/lib/codemirror.css");
 require("codemirror/mode/javascript/javascript");
 require("codemirror/mode/htmlmixed/htmlmixed");
 require("../lib/codemirror-mozilla.css");
-// enable to get local search to work
-// require("codemirror/addon/search/search");
-// require("codemirror/addon/search/searchcursor");
-// require("codemirror/addon/search/match-highliter");
```

Comment:
You could guard these with this `if` statement:

```js
if (process.env.TARGET !== "firefox-panel") {
  ...
}
```

As far as I know, webpack can see at build-time that it's false in the panel so it won't include these, but locally it will.